services/ui_backend_service/data/cache/generate_dag_action.py

```diff
@@ -134,8 +112,3 @@ def generate_dag(flow_id, tarball_path):
             'doc': node.doc
         }
     return dag
-
-
-class UnsupportedFlowLanguage(Exception):
-    def __str__(self):
-        return "Parsing DAG graph is not supported for the language used in this Flow."
```

Comment:
did this issue get alleviated by using the metaflow client? were you able to generate a DAG for a flow written with R?